addons/website/static/src/scss/website.scss

```diff
@@ -1370,31 +1372,47 @@ $ribbon-padding: 100px;
     pointer-events: none;
 }
 
-.o_ribbon_right {
+@mixin o-ribbon-right() {
     @include o-position-absolute($top: 0, $right: 0);
     padding: 0.5rem $ribbon-padding;
     // 0.708 is 1 - cos(45deg)
     // Transforms are applied right-to-left
     // Cannot use matrix because of the use of % values.
     transform: translateX(calc(-0.708 * (100% - #{2 * $ribbon-padding}))) rotate(45deg) translateX(calc(100% - #{$ribbon-padding}));
     transform-origin: top right;
-}
+};
 
-.o_ribbon_left {
+@mixin o-ribbon-left() {
     @include o-position-absolute($top: 0, $left: 0);
     padding: 0.5rem $ribbon-padding;
     transform: translateX(calc(0.708 * (100% - #{2 * $ribbon-padding}) - 100%)) rotate(-45deg) translateX($ribbon-padding);
     transform-origin: top right;
-}
+};
 
-.o_tag_right {
+@mixin o-tag-right() {
     @include o-position-absolute($top: 0, $right: 0);
     padding: 0.25rem 1rem;
-}
+};
 
-.o_tag_left {
+@mixin o-tag-left() {
     @include o-position-absolute($top: 0, $left: 0);
     padding: 0.25rem 1rem;
```

Comment:
I am not sure that this is the right place to put the mixins but I did not find a file with the mixins as in web_editor